Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -234,6 +258,19 @@ namespace Babylon::Plugins
         });
     }
 
+    void NativeInput::Impl::PointerScroll(uint32_t pointerId, uint32_t scrollAxis, int32_t scrollValue, DeviceType deviceType)
+    {
+        m_runtimeScheduler([pointerId, scrollAxis, scrollValue, deviceType, this]()
+            {
+                std::vector<int32_t>& deviceInputs{GetOrCreateInputMap(deviceType, pointerId, {scrollAxis})};
+                SetInputState(deviceType, pointerId, scrollAxis, scrollValue, deviceInputs, true);
+
+                m_eventDispatcher.tick(arcana::cancellation::none());
+                SetInputState(deviceType, pointerId, scrollAxis, 0, deviceInputs, true);
```

Comment:
Why is this set back to 0?

Reply:
Mousewheel events don't have a "wheelup" event so if we don't zero it out once the frame is complete, anything that is polling or checking this value will continue to see this as a non-zero value and can keep applying it.  Because of this, we need to zero it out when the inputChanged callback is complete

Reply:
What is "wheelup" (or "wheeldown")? I thought this was for an axis change (e.g. scroll) as opposed to click? I must be missing something.

Reply:
What I mean by "wheelup" is just that when you trigger a wheelevent, there no defined end event.  Furthermore, because of how we handle input values and storing, we need a way to purge stale values.  Since these values should only be used via the input changed observable, we would just purge them at the next tick.  Basically, we turned wheel input into a sort of button.